### ISSUE TITLE
Plugin Usage: Warning: React version not specified in eslint-plugin-react settings.

### ISSUE THREAD

**Issue by bhardy** [yannickcr/eslint-plugin-react#1963]
I am getting a similar issue to (https://github.com/yannickcr/eslint-plugin-react/issues/1955, https://github.com/yannickcr/eslint-plugin-react/pull/1857) where I receive the warning (` Warning: React version not specified in eslint-plugin-react settings`) when I only add `react` to my `plugins` and not my `extends`. We only want to grab 2 rules from `react`. I have the settings object in my `eslintrc.json` with the `"settings": { "react": { "version" : "16.3.2" }}`, along with rules:
```
  "rules": {
    "semi": ["error", "always"],
    "eqeqeq": "off",
    "react/jsx-no-bind": "off",
    "react/no-deprecated": "error",
    "no-console": "error"
  }
```

If I put `react` (or `"plugin:react/recommended"` into `extends` I no longer receive the warning, but I don't want to extend the rules.

Please let me know if this is an issue or if I'm doing something wrong!

Thank you

**Comment by nikolas** [comment]
I'm getting this warning as well, even with `plugin:react/recommended` in `extends`. Here's my `.eslintrc.json`:

```
{
    "extends": ["eslint:recommended", "plugin:react/recommended"],
    "parser": "babel-eslint",
    "parserOptions": {
        "ecmaVersion": 6,
        "sourceType": "module",
        "ecmaFeatures": {
            "jsx": true
        }
    },
    "plugins": [
        "react"
    ],
    "env": {
        "browser": true,
        "es6": true
    },
    "rules": {
        "no-console": "off"
    }
}
```

The page this error links to (https://github.com/yannickcr/eslint-plugin-react#configuration) doesn't mention anything about specifying the React version in the eslint config. Should this warning just be removed?

**Comment by ljharb** [comment]
@nikolas it definitely does; it’s listed there under “detect”. Regardless, if it didn’t, we’d need to fix the docs, not remove the warning.

I believe this issue should be fixed now in v7.12.4; closing.

Please file a new issue if it’s not.
